src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsDataRepairResource.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.prisonertonomisupdate.adjudications
+
+import com.microsoft.applicationinsights.TelemetryClient
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.http.HttpStatus
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class AdjudicationsDataRepairResource(
+  private val adjudicationService: AdjudicationsService,
+  private val telemetryClient: TelemetryClient,
+) {
+  @PostMapping("/prisons/{prisonId}/prisoners/{offenderNo}/adjudication/dps-charge-number/{chargeNumber}/punishments/repair")
+  @ResponseStatus(HttpStatus.OK)
+  @PreAuthorize("hasRole('NOMIS_ADJUDCIATIONS')")
```

Comment:
Does this need the `ROLE_` prefix?

Reply:
Guess not if the test works.

Reply:
yeah, using @steverendell convention of no ROLE from now on